operator-common/src/main/java/io/strimzi/operator/common/InformerUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common;
+
+import io.fabric8.kubernetes.client.informers.SharedIndexInformer;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Utilities for working with informers
+ */
+public class InformerUtils {
+    private static final ReconciliationLogger LOGGER = ReconciliationLogger.create(InformerUtils.class);
+
+    /**
+     * Logs exceptions in the informers to give us a better overview of what is happening.
+     *
+     * @param type          Type of the informer
+     * @param isStarted     Flag indicating whether the informer is already started
+     * @param throwable     Throwable describing the exception which occurred
+     *
+     * @return  Boolean indicating whether the informer should retry or not.
+     */
+    public static boolean loggingExceptionHandler(String type, boolean isStarted, Throwable throwable) {
+        LOGGER.warnOp("Caught exception in the " + type + " informer which is " + (isStarted ? "started" : "not started"), throwable);
```

Comment:
If no exception is expected and this exception indicates the non-delivery of an event it would seem to me that error, rather than warn, is more appropriate.